examples/activities/buildvrt.js

```diff
@@ -1,14 +1,15 @@
 "use strict";
```

Comment:
:+1: on externalizing all activities (with the possible exception of `echo`)

Once we have a clear mechanism for loading external activities, we should package them separately.